SSO-Auth/Config/configPage.html

```diff
@@ -1,15 +1,272 @@
+                  administrator will add permissions. The permissions of
+                  existing users will not be rewritten on subsequent logins.
+                </div>
+              </div>
+
+              <div class="checkboxContainer checkboxContainer-withDescription">
+                <label>
+                  <input
+                    id="EnableAllFolders"
+                    name="EnableAllFolders"
+                    type="checkbox"
+                  />
+                  <span>Enable All Folders</span>
+                </label>
+                <div class="fieldDescription checkboxFieldDescription">
+                  Determines if the user logging in is allowed access to all
+                  folders.
+                </div>
+              </div>
+
+              <!-- Make this multiline, one per line and then split on newline -->
+
+              <div class="inputContainer">
+                <label
+                  class="inputLabel inputLabelUnfocused"
+                  for="EnabledFolders"
+                  >Enabled Folders:</label
+                >
+                <input is="emby-input" id="EnabledFolders" type="text" />
+                <div class="fieldDescription">
+                  JSON Array of Strings. If <strong>enableAllFolders</strong> is
+                  set to false, then this will be used to determine what folders
+                  the users who log in through this provider are allowed to use.
+                </div>
+              </div>
+
+              <div class="inputContainer">
+                <label class="inputLabel inputLabelUnfocused" for="Roles"
+                  >Roles:</label
+                >
+                <input is="emby-input" id="Roles" type="text" />
+                <div class="fieldDescription">
+                  This validates the OpenID response against the claim set in
+                  RoleClaim. If a user has any of these roles, then the user is
+                  authenticated. Leave blank to disable role checking. This
+                  currently only works for Keycloak (to my knowledge).
+                </div>
+              </div>
+
+              <div class="inputContainer">
+                <label class="inputLabel inputLabelUnfocused" for="AdminRoles"
+                  >Admin Roles:</label
+                >
+                <input is="emby-input" id="AdminRoles" type="text" />
+                <div class="fieldDescription">
+                  Like "Roles", but confers admin privelege. If unset will not
+                  grant admin priveleges.
+                </div>
+              </div>
+
+              <div class="checkboxContainer checkboxContainer-withDescription">
+                <label>
+                  <input
+                    id="EnableFolderRoles"
+                    name="EnableFolderRoles"
+                    type="checkbox"
+                  />
+                  <span>Enable role-based folders</span>
```

Comment:
Consistency. Should be something like "Enable Role-Based Folder Access:"

Reply:
Resolved by https://github.com/9p4/jellyfin-plugin-sso/pull/18/commits/5112bb13afe9943aad8784e551d7277870ddeaae